playwright.config.ts

```diff
@@ -0,0 +1,106 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: "./tests",
```

Comment:
Is tests confusing regarding the type of tests? Unit, intergration, e2e? Maybe a sub-folder would work?